homeassistant/components/climate/device_action.py

```diff
@@ -42,7 +42,18 @@
     }
 )
 
-_ACTION_SCHEMA = vol.Any(SET_HVAC_MODE_SCHEMA, SET_PRESET_MODE_SCHEMA)
+SET_TEMPERATURE_SCHEMA = cv.DEVICE_ACTION_BASE_SCHEMA.extend(
```

Comment:
Seems temperature is missing in the schema?

Reply:
done